nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/jquery/modal/jquery.modal.js

```diff
@@ -50,6 +50,7 @@
  *          disabled: isDisabledFunction,
  *      handler: {
  *          click: applyHandler
+ *          keyup: keyupHandler
```

Comment:
Is there a reason the `keyupHandler` is specified per button? If both Apply and Cancel have keyupHandlers specified, which one is called? Can one handler 'consume' the event and prevent further listeners from triggering? 

Does it make sense to only allow a single key listener per modal? In other places, we have placed key listeners on components within a dialog. Was that not an option here? How do the key listeners effect those if configured at the same time?